### DISCUSSION TITLE
Getting current weather from a gridpoint?

### DISCUSSION THREAD

**Discussion by Whamodyne** [weather-gov/api#667] [Q&A]
All,  

I know I can get current weather from observation stations, but can I do it somehow from a gridpoint?

I can get a forecast for my gridpoint of interest from https://api.weather.gov/gridpoints/PBZ/52,29/forecast

I want to know what the current on ground weather is for that gridpoint right now.  
Is it raining right now?
Is it snowing right now?
How hard and which direction is the wind blowing?

this gridpoint is miles and miles away from an observation station with many valleys and hills in between - lots of microclimates, so any observation from 30 miles away is not that relevant. 

I know that the radar covering this area can tell if its raining or not, is there anyway to get this info and not just a forecast for a gridpoint?  

I've been banging my head against the API and it doesn't look good but maybe I missed something. 

Thanks!

**Comment by dwhitemv25** [comment]
You can call [`/gridpoints/{wfo}/{x},{y}/stations`](https://www.weather.gov/documentation/services-web-api#/default/gridpoint_stations) to get a list of stations near the gridpoint. Whether that is the weather in that specific gridpoint or not, the API does not provide that data unless there is an observation station in that gridpoint. 

Other sources that are claiming to give the weather in a grid point or geographic point are using a weather model and/or private sensor network to provide synthesized data. Whether those conditions are correct or not depends on how much you trust their inputs.

Determining precipitation type (rain vs. snow) is difficult, even with specialized equipment. I suspect most places assume snow if the temperature/wet bulb are low enough, or use radar analysis. Even then it's a crap shoot. At my house in west-central Oregon we get a lot more snow than points a mile east get due to spillover off of the nearby Coast Range. In many cases WU/AerisWeather/etc. are claiming 'Rain' for the current condition, when there is white stuff falling outside. Go up the road a few thousand yards and no snow.

**Comment by matttproud** [comment]
https://github.com/weather-gov/api/discussions/661 may be helpful for ascertaining local current conditions.
